src/components/ReviewsList.jsx

```diff
@@ -1,41 +1,65 @@
 import { useEffect, useState } from "react";
 import { fetchReviews } from "../utils/utils";
-import "../css/ReviewsList.css"
-import { Link, useSearchParams } from "react-router-dom";
+import "../css/ReviewsList.css";
+import { Link, useNavigate, useSearchParams } from "react-router-dom";
 
 export default function ReviewsList() {
 	const [reviews, setReviews] = useState(false);
-    const [params, setParams ] = useSearchParams()
-    const categoriesQuery = params.get("category") ?? undefined
-
+	const [params, setParams] = useSearchParams();
```

Comment:
We also want to add a loading state here, so that if the data is still loading we can display a loading message or icon. :)

https://notes.northcoders.com/courses/js-front-end/useEffect#loading-patterns

Reply:
i have done

Reply:
im using the reviews state

Reply:
line 28